docs/source/user/workspaces.rst

```diff
@@ -0,0 +1,115 @@
+.. Copyright (c) Jupyter Development Team.
+.. Distributed under the terms of the Modified BSD License.
+
+.. _workspaces:
+
+Workspaces
+==========
+
+A JupyterLab Workspace defines the layout and state of the user interface such as the position of files, notebooks, sidebars, and open/closed state of the panels.
```

Comment:
Comment from @JasonWeill which I mistakenly deleted while preparing my reply to it :facepalm: 

> We should indicate that a workspace name may only contain ASCII letters a-z, A-Z, and digits 0-9 (regex "word" characters) and hyphens. [#14340](https://github.com/jupyterlab/jupyterlab/issues/14340) was a bug related to Chinese text handling because we incorrectly relied on "word" characters being used, exclusively, globally, in filenames.

Reply:
Done in 6459bc05f21581966736ee13d754b50dc9904bf6.